main.js

```diff
@@ -3,9 +3,13 @@
  * https://github.com/marketplace/actions/cloudflare-create-dns-record
  */
 
-const path = require("path");
 const cp = require("child_process");
 
+const saveOutput = ({ id, name }) => {
+  console.log(`echo "id=${id}" >> $GITHUB_OUTPUT`);
+  console.log(`echo "name=${name}" >> $GITHUB_OUTPUT`);
+};
```

Comment:
Thoughts on adding a 3rd output option here whether the record was created or updated?
We could then key off of the upsert and conditionally perform other actions

Reply:
Sounds useful to me!

Reply:
Added in 54aae4f